packages/apps-config/src/endpoints/productionRelayPolkadot.ts

```diff
@@ -170,7 +170,6 @@ export const prodParasPolkadot: EndpointOption[] = [
     providers: {
       Efinity: 'wss://rpc.efinity.io',
       Dwellir: 'wss://efinity-rpc.dwellir.com',
```

Comment:
```suggestion
      Dwellir: 'wss://efinity-rpc.dwellir.com'
```

Just remove the trailing , (rest all all-ok)